tests/move/arguments/sources/multi_signer.move

```diff
@@ -0,0 +1,49 @@
+script {
+    use std::signer;
+    use std::object::{Object};
+    use std::string::{String};
+    use transaction_arguments::tx_args_module::{Self, EmptyResource};
+
+    fun main(
```

Comment:
You can actually name this any name, and that'll be the name of the script bytecode file.